corehq/apps/smsforms/app.py

```diff
@@ -56,9 +61,20 @@ def start_session(domain, contact, app, module, form, case_id=None, yield_respon
         }
     
     if app and form:
```

Comment:
@kaapstorm Sorry I should have mentioned sometimes the tests fail for some people because of something funny with couch where there needs to be a delay before querying for something after saving it. I ran all the manual sms and ivr tests and they passed, so no need to rerun it, but I'm guessing if you add sleep(1) (or some bigger number) in these places it would pass for you:

https://github.com/dimagi/commcare-hq/blob/9baeaa823a47063f958f30c380fa43ea3e0ad97a/corehq/apps/sms/tests/util.py#L240
https://github.com/dimagi/commcare-hq/blob/9baeaa823a47063f958f30c380fa43ea3e0ad97a/corehq/apps/sms/tests/util.py#L263